src/ape/managers/networks.py

```diff
@@ -46,6 +46,59 @@ def active_provider(self) -> Optional[ProviderAPI]:
     @active_provider.setter
     def active_provider(self, new_value: ProviderAPI):
         self._active_provider = new_value
+    
+    @property
+    def network(self) -> NetworkAPI:
+        """
+        The current network if connected to one.
+
+        Raises:
+            :class:`~ape.exceptions.ProviderNotConnectedError`: When there is
+            no active provider at runtime.
+
+        Returns:
+            :class:`~ape.api.networks.NetworkAPI`
+        """
+        return self.provider.network
+    
+    @property
+    def ecosystem(self) -> EcosystemAPI:
+        """
+        The current ecosystem if connected to one.
+
+        Raises:
+            :class:`~ape.exceptions.ProviderNotConnectedError`: When there is
+            no active provider at runtime.
+
+        Returns:
+            :class:`~ape.api.providers.ProviderAPI`
+        """
+        return self.network.ecosystem
+
+    def fork(
+        self,
+        provider_name: Optional[str] = None,
+        provider_settings: Optional[Dict] = None,
+    ) -> ProviderContextManager:
+        """
+        Fork the currently connected network.
+
+        Args:
+            provider_name (str, optional): The name of the provider to get. Defaults to ``None``.
+              When ``None``, returns the default provider.
+            provider_settings (dict, optional): Settings to apply to the provider. Defaults to
+              ``None``.
+
+        Returns:
+            :class:`~ape.api.networks.ProviderContextManager`
+        """
+        forked_network = self.ecosystem.get_network(f"{self.network.name}-fork")
```

Comment:
IDK, this isn't too important, but it would allow "cloning" already forked networks:

```suggestion
        forked_network = self.ecosystem.get_network(f"{self.network.name.replace('fork', ''}-fork")
```

Reply:
It wouldn't be a clone though, because it would fork mainnet again and lose all of your fork history